webapp/app.py

```diff
@@ -18,3 +22,19 @@
 @app.route("/")
 def index():
     return render_template("index.html")
+
+
+url_prefix = "/docs"
+server_docs_parser = DocParser(
+    api=DiscourseAPI(base_url="https://discourse.ubuntu.com/"),
```

Comment:
Had a chat with @tbille and we feel it's better to override the cached session (which we're deprecating anyway) with a [Talisker session](https://talisker.readthedocs.io/en/latest/requests.html#using-a-talisker-session) instead, so we get API request latency metrics in the logs.

Could you please instantiate `DiscourseAPI` as follows:

``` python3
import talisker

# ...
    api=DiscourseAPI(
        base_url="https://discourse.ubuntu.com/",
        session=talisker.requests.get_session()
    ),
```

?

Reply:
Will do!